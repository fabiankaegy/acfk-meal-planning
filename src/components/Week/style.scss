.week {
	list-style: none;
	padding: 0;
	margin: 0;
	display: grid;
	grid-gap: 25px;
	width: 90%;
	margin: 25px auto;
	overflow-x: scroll;
	scroll-snap-type: x mandatory;
	scroll-snap-points-x: repeat(225px);
	grid-template-columns: repeat(1, 1fr);
	@media (min-width: 600px) {
		grid-template-columns: repeat(7, minmax(150px, 1fr));
	}
	> * {
		scroll-snap-align: start;
	}
}

.days-navgaiton {
	width: 100%;
	display: flex;
	justify-content: space-between;
	flex-direction: row;
	button {
		flex-grow: 1;
		padding: 10px;
		border: 0;
		background: #f8f8f8;
		border-color: #cccccc;
		border-left: 1px;
		border-right: 1px;
		&:hover {
			filter: brightness(85%);
			cursor: pointer;
		}
		&.active {
			filter: brightness(85%);
		}
	}
}
