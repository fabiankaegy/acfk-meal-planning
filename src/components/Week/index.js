import React from 'react';
import Day from '../Day';
import './style.scss';

const Week = props => {
	const days = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday'];

	return (
		<section className="week">
			{days.map((name, key) => (
				<Day key={key} title={name} />
			))}
		</section>
	);
};

export default Week;
