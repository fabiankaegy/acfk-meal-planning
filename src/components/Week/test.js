import React from "react";
import ReactDOM from "react-dom";
import { render, cleanup } from "react-testing-library";
import Week from "./index";

describe("Week Component", function() {
  let wrapper;

  beforeEach(function() {
    wrapper = render(<Week />);
  });

  it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<Week />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it("matches snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
