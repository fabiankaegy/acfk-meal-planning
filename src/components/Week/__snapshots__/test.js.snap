// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Week Component matches snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        class="week"
      >
        <div
          class="day"
        >
          <header
            class="header"
          >
            <h2>
              Monday
            </h2>
          </header>
          <div
            class="recipes"
            data-testid="recipes"
          >
            <div
              class="button-wrapper"
            >
              <button
                class="button"
              >
                <svg
                  height="41.511"
                  viewBox="0 0 41.511 41.511"
                  width="41.511"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <g
                    data-name="Gruppe 1"
                    fill="none"
                    id="Gruppe_1"
                    stroke="#707070"
                    stroke-linecap="round"
                    stroke-width="5"
                    transform="translate(2.5 2.5)"
                  >
                    <line
                      data-name="Linie 1"
                      id="Linie_1"
                      transform="translate(18.255)"
                      y2="36.511"
                    />
                    <line
                      data-name="Linie 2"
                      id="Linie_2"
                      transform="rotate(90 9.128 27.383)"
                      y2="36.511"
                    />
                  </g>
                </svg>
              </button>
            </div>
          </div>
        </div>
        <div
          class="day"
        >
          <header
            class="header"
          >
            <h2>
              Tuesday
            </h2>
          </header>
          <div
            class="recipes"
            data-testid="recipes"
          >
            <div
              class="button-wrapper"
            >
              <button
                class="button"
              >
                <svg
                  height="41.511"
                  viewBox="0 0 41.511 41.511"
                  width="41.511"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <g
                    data-name="Gruppe 1"
                    fill="none"
                    id="Gruppe_1"
                    stroke="#707070"
                    stroke-linecap="round"
                    stroke-width="5"
                    transform="translate(2.5 2.5)"
                  >
                    <line
                      data-name="Linie 1"
                      id="Linie_1"
                      transform="translate(18.255)"
                      y2="36.511"
                    />
                    <line
                      data-name="Linie 2"
                      id="Linie_2"
                      transform="rotate(90 9.128 27.383)"
                      y2="36.511"
                    />
                  </g>
                </svg>
              </button>
            </div>
          </div>
        </div>
        <div
          class="day"
        >
          <header
            class="header"
          >
            <h2>
              Wednesday
            </h2>
          </header>
          <div
            class="recipes"
            data-testid="recipes"
          >
            <div
              class="button-wrapper"
            >
              <button
                class="button"
              >
                <svg
                  height="41.511"
                  viewBox="0 0 41.511 41.511"
                  width="41.511"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <g
                    data-name="Gruppe 1"
                    fill="none"
                    id="Gruppe_1"
                    stroke="#707070"
                    stroke-linecap="round"
                    stroke-width="5"
                    transform="translate(2.5 2.5)"
                  >
                    <line
                      data-name="Linie 1"
                      id="Linie_1"
                      transform="translate(18.255)"
                      y2="36.511"
                    />
                    <line
                      data-name="Linie 2"
                      id="Linie_2"
                      transform="rotate(90 9.128 27.383)"
                      y2="36.511"
                    />
                  </g>
                </svg>
              </button>
            </div>
          </div>
        </div>
        <div
          class="day"
        >
          <header
            class="header"
          >
            <h2>
              Thursday
            </h2>
          </header>
          <div
            class="recipes"
            data-testid="recipes"
          >
            <div
              class="button-wrapper"
            >
              <button
                class="button"
              >
                <svg
                  height="41.511"
                  viewBox="0 0 41.511 41.511"
                  width="41.511"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <g
                    data-name="Gruppe 1"
                    fill="none"
                    id="Gruppe_1"
                    stroke="#707070"
                    stroke-linecap="round"
                    stroke-width="5"
                    transform="translate(2.5 2.5)"
                  >
                    <line
                      data-name="Linie 1"
                      id="Linie_1"
                      transform="translate(18.255)"
                      y2="36.511"
                    />
                    <line
                      data-name="Linie 2"
                      id="Linie_2"
                      transform="rotate(90 9.128 27.383)"
                      y2="36.511"
                    />
                  </g>
                </svg>
              </button>
            </div>
          </div>
        </div>
        <div
          class="day"
        >
          <header
            class="header"
          >
            <h2>
              Friday
            </h2>
          </header>
          <div
            class="recipes"
            data-testid="recipes"
          >
            <div
              class="button-wrapper"
            >
              <button
                class="button"
              >
                <svg
                  height="41.511"
                  viewBox="0 0 41.511 41.511"
                  width="41.511"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <g
                    data-name="Gruppe 1"
                    fill="none"
                    id="Gruppe_1"
                    stroke="#707070"
                    stroke-linecap="round"
                    stroke-width="5"
                    transform="translate(2.5 2.5)"
                  >
                    <line
                      data-name="Linie 1"
                      id="Linie_1"
                      transform="translate(18.255)"
                      y2="36.511"
                    />
                    <line
                      data-name="Linie 2"
                      id="Linie_2"
                      transform="rotate(90 9.128 27.383)"
                      y2="36.511"
                    />
                  </g>
                </svg>
              </button>
            </div>
          </div>
        </div>
        <div
          class="day"
        >
          <header
            class="header"
          >
            <h2>
              Saturday
            </h2>
          </header>
          <div
            class="recipes"
            data-testid="recipes"
          >
            <div
              class="button-wrapper"
            >
              <button
                class="button"
              >
                <svg
                  height="41.511"
                  viewBox="0 0 41.511 41.511"
                  width="41.511"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <g
                    data-name="Gruppe 1"
                    fill="none"
                    id="Gruppe_1"
                    stroke="#707070"
                    stroke-linecap="round"
                    stroke-width="5"
                    transform="translate(2.5 2.5)"
                  >
                    <line
                      data-name="Linie 1"
                      id="Linie_1"
                      transform="translate(18.255)"
                      y2="36.511"
                    />
                    <line
                      data-name="Linie 2"
                      id="Linie_2"
                      transform="rotate(90 9.128 27.383)"
                      y2="36.511"
                    />
                  </g>
                </svg>
              </button>
            </div>
          </div>
        </div>
        <div
          class="day"
        >
          <header
            class="header"
          >
            <h2>
              Sunday
            </h2>
          </header>
          <div
            class="recipes"
            data-testid="recipes"
          >
            <div
              class="button-wrapper"
            >
              <button
                class="button"
              >
                <svg
                  height="41.511"
                  viewBox="0 0 41.511 41.511"
                  width="41.511"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <g
                    data-name="Gruppe 1"
                    fill="none"
                    id="Gruppe_1"
                    stroke="#707070"
                    stroke-linecap="round"
                    stroke-width="5"
                    transform="translate(2.5 2.5)"
                  >
                    <line
                      data-name="Linie 1"
                      id="Linie_1"
                      transform="translate(18.255)"
                      y2="36.511"
                    />
                    <line
                      data-name="Linie 2"
                      id="Linie_2"
                      transform="rotate(90 9.128 27.383)"
                      y2="36.511"
                    />
                  </g>
                </svg>
              </button>
            </div>
          </div>
        </div>
      </section>
    </div>
    <div>
      <section
        class="week"
      >
        <div
          class="day"
        >
          <header
            class="header"
          >
            <h2>
              Monday
            </h2>
          </header>
          <div
            class="recipes"
            data-testid="recipes"
          >
            <div
              class="button-wrapper"
            >
              <button
                class="button"
              >
                <svg
                  height="41.511"
                  viewBox="0 0 41.511 41.511"
                  width="41.511"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <g
                    data-name="Gruppe 1"
                    fill="none"
                    id="Gruppe_1"
                    stroke="#707070"
                    stroke-linecap="round"
                    stroke-width="5"
                    transform="translate(2.5 2.5)"
                  >
                    <line
                      data-name="Linie 1"
                      id="Linie_1"
                      transform="translate(18.255)"
                      y2="36.511"
                    />
                    <line
                      data-name="Linie 2"
                      id="Linie_2"
                      transform="rotate(90 9.128 27.383)"
                      y2="36.511"
                    />
                  </g>
                </svg>
              </button>
            </div>
          </div>
        </div>
        <div
          class="day"
        >
          <header
            class="header"
          >
            <h2>
              Tuesday
            </h2>
          </header>
          <div
            class="recipes"
            data-testid="recipes"
          >
            <div
              class="button-wrapper"
            >
              <button
                class="button"
              >
                <svg
                  height="41.511"
                  viewBox="0 0 41.511 41.511"
                  width="41.511"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <g
                    data-name="Gruppe 1"
                    fill="none"
                    id="Gruppe_1"
                    stroke="#707070"
                    stroke-linecap="round"
                    stroke-width="5"
                    transform="translate(2.5 2.5)"
                  >
                    <line
                      data-name="Linie 1"
                      id="Linie_1"
                      transform="translate(18.255)"
                      y2="36.511"
                    />
                    <line
                      data-name="Linie 2"
                      id="Linie_2"
                      transform="rotate(90 9.128 27.383)"
                      y2="36.511"
                    />
                  </g>
                </svg>
              </button>
            </div>
          </div>
        </div>
        <div
          class="day"
        >
          <header
            class="header"
          >
            <h2>
              Wednesday
            </h2>
          </header>
          <div
            class="recipes"
            data-testid="recipes"
          >
            <div
              class="button-wrapper"
            >
              <button
                class="button"
              >
                <svg
                  height="41.511"
                  viewBox="0 0 41.511 41.511"
                  width="41.511"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <g
                    data-name="Gruppe 1"
                    fill="none"
                    id="Gruppe_1"
                    stroke="#707070"
                    stroke-linecap="round"
                    stroke-width="5"
                    transform="translate(2.5 2.5)"
                  >
                    <line
                      data-name="Linie 1"
                      id="Linie_1"
                      transform="translate(18.255)"
                      y2="36.511"
                    />
                    <line
                      data-name="Linie 2"
                      id="Linie_2"
                      transform="rotate(90 9.128 27.383)"
                      y2="36.511"
                    />
                  </g>
                </svg>
              </button>
            </div>
          </div>
        </div>
        <div
          class="day"
        >
          <header
            class="header"
          >
            <h2>
              Thursday
            </h2>
          </header>
          <div
            class="recipes"
            data-testid="recipes"
          >
            <div
              class="button-wrapper"
            >
              <button
                class="button"
              >
                <svg
                  height="41.511"
                  viewBox="0 0 41.511 41.511"
                  width="41.511"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <g
                    data-name="Gruppe 1"
                    fill="none"
                    id="Gruppe_1"
                    stroke="#707070"
                    stroke-linecap="round"
                    stroke-width="5"
                    transform="translate(2.5 2.5)"
                  >
                    <line
                      data-name="Linie 1"
                      id="Linie_1"
                      transform="translate(18.255)"
                      y2="36.511"
                    />
                    <line
                      data-name="Linie 2"
                      id="Linie_2"
                      transform="rotate(90 9.128 27.383)"
                      y2="36.511"
                    />
                  </g>
                </svg>
              </button>
            </div>
          </div>
        </div>
        <div
          class="day"
        >
          <header
            class="header"
          >
            <h2>
              Friday
            </h2>
          </header>
          <div
            class="recipes"
            data-testid="recipes"
          >
            <div
              class="button-wrapper"
            >
              <button
                class="button"
              >
                <svg
                  height="41.511"
                  viewBox="0 0 41.511 41.511"
                  width="41.511"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <g
                    data-name="Gruppe 1"
                    fill="none"
                    id="Gruppe_1"
                    stroke="#707070"
                    stroke-linecap="round"
                    stroke-width="5"
                    transform="translate(2.5 2.5)"
                  >
                    <line
                      data-name="Linie 1"
                      id="Linie_1"
                      transform="translate(18.255)"
                      y2="36.511"
                    />
                    <line
                      data-name="Linie 2"
                      id="Linie_2"
                      transform="rotate(90 9.128 27.383)"
                      y2="36.511"
                    />
                  </g>
                </svg>
              </button>
            </div>
          </div>
        </div>
        <div
          class="day"
        >
          <header
            class="header"
          >
            <h2>
              Saturday
            </h2>
          </header>
          <div
            class="recipes"
            data-testid="recipes"
          >
            <div
              class="button-wrapper"
            >
              <button
                class="button"
              >
                <svg
                  height="41.511"
                  viewBox="0 0 41.511 41.511"
                  width="41.511"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <g
                    data-name="Gruppe 1"
                    fill="none"
                    id="Gruppe_1"
                    stroke="#707070"
                    stroke-linecap="round"
                    stroke-width="5"
                    transform="translate(2.5 2.5)"
                  >
                    <line
                      data-name="Linie 1"
                      id="Linie_1"
                      transform="translate(18.255)"
                      y2="36.511"
                    />
                    <line
                      data-name="Linie 2"
                      id="Linie_2"
                      transform="rotate(90 9.128 27.383)"
                      y2="36.511"
                    />
                  </g>
                </svg>
              </button>
            </div>
          </div>
        </div>
        <div
          class="day"
        >
          <header
            class="header"
          >
            <h2>
              Sunday
            </h2>
          </header>
          <div
            class="recipes"
            data-testid="recipes"
          >
            <div
              class="button-wrapper"
            >
              <button
                class="button"
              >
                <svg
                  height="41.511"
                  viewBox="0 0 41.511 41.511"
                  width="41.511"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <g
                    data-name="Gruppe 1"
                    fill="none"
                    id="Gruppe_1"
                    stroke="#707070"
                    stroke-linecap="round"
                    stroke-width="5"
                    transform="translate(2.5 2.5)"
                  >
                    <line
                      data-name="Linie 1"
                      id="Linie_1"
                      transform="translate(18.255)"
                      y2="36.511"
                    />
                    <line
                      data-name="Linie 2"
                      id="Linie_2"
                      transform="rotate(90 9.128 27.383)"
                      y2="36.511"
                    />
                  </g>
                </svg>
              </button>
            </div>
          </div>
        </div>
      </section>
    </div>
  </body>,
  "container": <div>
    <section
      class="week"
    >
      <div
        class="day"
      >
        <header
          class="header"
        >
          <h2>
            Monday
          </h2>
        </header>
        <div
          class="recipes"
          data-testid="recipes"
        >
          <div
            class="button-wrapper"
          >
            <button
              class="button"
            >
              <svg
                height="41.511"
                viewBox="0 0 41.511 41.511"
                width="41.511"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  data-name="Gruppe 1"
                  fill="none"
                  id="Gruppe_1"
                  stroke="#707070"
                  stroke-linecap="round"
                  stroke-width="5"
                  transform="translate(2.5 2.5)"
                >
                  <line
                    data-name="Linie 1"
                    id="Linie_1"
                    transform="translate(18.255)"
                    y2="36.511"
                  />
                  <line
                    data-name="Linie 2"
                    id="Linie_2"
                    transform="rotate(90 9.128 27.383)"
                    y2="36.511"
                  />
                </g>
              </svg>
            </button>
          </div>
        </div>
      </div>
      <div
        class="day"
      >
        <header
          class="header"
        >
          <h2>
            Tuesday
          </h2>
        </header>
        <div
          class="recipes"
          data-testid="recipes"
        >
          <div
            class="button-wrapper"
          >
            <button
              class="button"
            >
              <svg
                height="41.511"
                viewBox="0 0 41.511 41.511"
                width="41.511"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  data-name="Gruppe 1"
                  fill="none"
                  id="Gruppe_1"
                  stroke="#707070"
                  stroke-linecap="round"
                  stroke-width="5"
                  transform="translate(2.5 2.5)"
                >
                  <line
                    data-name="Linie 1"
                    id="Linie_1"
                    transform="translate(18.255)"
                    y2="36.511"
                  />
                  <line
                    data-name="Linie 2"
                    id="Linie_2"
                    transform="rotate(90 9.128 27.383)"
                    y2="36.511"
                  />
                </g>
              </svg>
            </button>
          </div>
        </div>
      </div>
      <div
        class="day"
      >
        <header
          class="header"
        >
          <h2>
            Wednesday
          </h2>
        </header>
        <div
          class="recipes"
          data-testid="recipes"
        >
          <div
            class="button-wrapper"
          >
            <button
              class="button"
            >
              <svg
                height="41.511"
                viewBox="0 0 41.511 41.511"
                width="41.511"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  data-name="Gruppe 1"
                  fill="none"
                  id="Gruppe_1"
                  stroke="#707070"
                  stroke-linecap="round"
                  stroke-width="5"
                  transform="translate(2.5 2.5)"
                >
                  <line
                    data-name="Linie 1"
                    id="Linie_1"
                    transform="translate(18.255)"
                    y2="36.511"
                  />
                  <line
                    data-name="Linie 2"
                    id="Linie_2"
                    transform="rotate(90 9.128 27.383)"
                    y2="36.511"
                  />
                </g>
              </svg>
            </button>
          </div>
        </div>
      </div>
      <div
        class="day"
      >
        <header
          class="header"
        >
          <h2>
            Thursday
          </h2>
        </header>
        <div
          class="recipes"
          data-testid="recipes"
        >
          <div
            class="button-wrapper"
          >
            <button
              class="button"
            >
              <svg
                height="41.511"
                viewBox="0 0 41.511 41.511"
                width="41.511"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  data-name="Gruppe 1"
                  fill="none"
                  id="Gruppe_1"
                  stroke="#707070"
                  stroke-linecap="round"
                  stroke-width="5"
                  transform="translate(2.5 2.5)"
                >
                  <line
                    data-name="Linie 1"
                    id="Linie_1"
                    transform="translate(18.255)"
                    y2="36.511"
                  />
                  <line
                    data-name="Linie 2"
                    id="Linie_2"
                    transform="rotate(90 9.128 27.383)"
                    y2="36.511"
                  />
                </g>
              </svg>
            </button>
          </div>
        </div>
      </div>
      <div
        class="day"
      >
        <header
          class="header"
        >
          <h2>
            Friday
          </h2>
        </header>
        <div
          class="recipes"
          data-testid="recipes"
        >
          <div
            class="button-wrapper"
          >
            <button
              class="button"
            >
              <svg
                height="41.511"
                viewBox="0 0 41.511 41.511"
                width="41.511"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  data-name="Gruppe 1"
                  fill="none"
                  id="Gruppe_1"
                  stroke="#707070"
                  stroke-linecap="round"
                  stroke-width="5"
                  transform="translate(2.5 2.5)"
                >
                  <line
                    data-name="Linie 1"
                    id="Linie_1"
                    transform="translate(18.255)"
                    y2="36.511"
                  />
                  <line
                    data-name="Linie 2"
                    id="Linie_2"
                    transform="rotate(90 9.128 27.383)"
                    y2="36.511"
                  />
                </g>
              </svg>
            </button>
          </div>
        </div>
      </div>
      <div
        class="day"
      >
        <header
          class="header"
        >
          <h2>
            Saturday
          </h2>
        </header>
        <div
          class="recipes"
          data-testid="recipes"
        >
          <div
            class="button-wrapper"
          >
            <button
              class="button"
            >
              <svg
                height="41.511"
                viewBox="0 0 41.511 41.511"
                width="41.511"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  data-name="Gruppe 1"
                  fill="none"
                  id="Gruppe_1"
                  stroke="#707070"
                  stroke-linecap="round"
                  stroke-width="5"
                  transform="translate(2.5 2.5)"
                >
                  <line
                    data-name="Linie 1"
                    id="Linie_1"
                    transform="translate(18.255)"
                    y2="36.511"
                  />
                  <line
                    data-name="Linie 2"
                    id="Linie_2"
                    transform="rotate(90 9.128 27.383)"
                    y2="36.511"
                  />
                </g>
              </svg>
            </button>
          </div>
        </div>
      </div>
      <div
        class="day"
      >
        <header
          class="header"
        >
          <h2>
            Sunday
          </h2>
        </header>
        <div
          class="recipes"
          data-testid="recipes"
        >
          <div
            class="button-wrapper"
          >
            <button
              class="button"
            >
              <svg
                height="41.511"
                viewBox="0 0 41.511 41.511"
                width="41.511"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  data-name="Gruppe 1"
                  fill="none"
                  id="Gruppe_1"
                  stroke="#707070"
                  stroke-linecap="round"
                  stroke-width="5"
                  transform="translate(2.5 2.5)"
                >
                  <line
                    data-name="Linie 1"
                    id="Linie_1"
                    transform="translate(18.255)"
                    y2="36.511"
                  />
                  <line
                    data-name="Linie 2"
                    id="Linie_2"
                    transform="rotate(90 9.128 27.383)"
                    y2="36.511"
                  />
                </g>
              </svg>
            </button>
          </div>
        </div>
      </div>
    </section>
  </div>,
  "debug": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllBySelectText": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getAllByValue": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getBySelectText": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "getByValue": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllBySelectText": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryAllByValue": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryBySelectText": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "queryByValue": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
